docker-compose.yml

```diff
@@ -89,6 +78,50 @@ services:
         target: /src
         read_only: true
 
+  seeddb:
+    container_name: drb_local_seed
+    depends_on:
+      rabbitmq:
+        condition: service_healthy
+      devsetup:
+        condition: service_completed_successfully
+      database:
+        condition: service_started
+      elasticsearch:
+        condition: service_started
+      redis:
+        condition: service_started
+      s3:
+        condition: service_started
```

Comment:
added all of these conditions bc otherwise it would sometimes fail when rabbitmq wasn't yet ready to accept connections or the devsetup wasn't completely finished